config.toml

```diff
@@ -203,15 +206,15 @@ package = "python-tornado"
 [origins.saltbundlepy-urllib3]
 project = "SUSE:SLE-15-SP4:Update"
 package = "python-urllib3"
-[origins.saltbundlepy-venvjail]
-project = "SUSE:SLE-15-SP4:Update"
-package = "python-venvjail"
```

Comment:
Likewise, `venvjail` is still used.

Reply:
It is used, but it is built from Github (as per https://github.com/openSUSE/salt/wiki/Salt-Bundle) - it has no upstream package in OBS, hence I removed it.